src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle {
+    private int length;
+    private int width;
+
+    public int getLength() {
+        return length;
+    }
+
+    public int getWidth() {
+        return width;
+    }
+
+    public void setLength(int length) {
+        this.length = length;
+    }
+
+    public void setWidth(int width) {
+        this.width = width;
+    }
+
+    public int getArea(){
+        return getLength() * getWidth();

```

Comment:
For better readability, instead of calling the getters inside the 'getArea' method, you can directly use the 'length' and 'width' fields since you are within the same class.